src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: space-between;
```

Comment:
This doesn't have any affect because this value for align-items doesn't exist. See [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)
```suggestion
```